app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -986,7 +986,13 @@ class BrowserTabFragment :
 
     override fun onResume() {
         super.onResume()
+
+        if (omnibar.omnibarPosition != settingsDataStore.omnibarPosition) {
```

Comment:
this logic belongs in the `ViewModel` so it can be tested

Reply:
Updated [here](https://github.com/duckduckgo/Android/pull/4930/commits/92b8ab1ff9e225f9e585c3ffd2da328811f5db08).